### ISSUE TITLE
Crash when using the nugget in Microsoft Azure Web Apps

### ISSUE THREAD

**Issue by Kreepounet** [googleads/googleads-dotnet-lib#75]
There is a bug in the code that causes Azure Web Apps to crash. After some reasearch, we found a solution. In fact, the problem appears when using cryptography with UserKeySet (default value). You have to use MachineKeySet instead. Here is the the way to correct the bug.

In OAuth2ProviderBase.cs, after line 368, add this :

`
      cspParam.Flags = CspProviderFlags.UseMachineKeyStore;
`

In OAuth2ProviderForServiceAccountsCustom.cs, replace lines 145/146 by :

```
X509Certificate2 jwtCertificate = new X509Certificate2(JwtCertificatePath,
          JwtCertificatePassword, X509KeyStorageFlags.MachineKeySet);
```

I don't know if there are other changes to make because we only use service accounts.

**Comment by AnashOommen** [comment]
We are looking into this issue, thanks for identifying the fix.

**Comment by AnashOommen** [comment]
This should be fixed as of v18.26.0. The new code is based on BouncyCastle, and hence no machinekeyset / store are involved.
